chpl/chpl-resources/src/main/resources/errors.properties

```diff
@@ -187,23 +187,23 @@ listing.criteria.testFunctionalityNotFoundAndRemoved=Criteria %s contains an inv
 listing.criteria.testToolNotFoundAndRemoved=Criteria %s contains an invalid test tool '%s'. It has been removed from the pending listing.
 
 listing.uniqueId.maxLength=You have exceeded the max length, %s characters, for the Unique CHPL ID %s.
-listing.vendorName.maxlength=You have exceeded the max length, %s characters, for the Vendor Name %s.
+listing.DeveloperName.maxlength=You have exceeded the max length, %s characters, for the Developer Name %s.
 listing.productName.maxlength=You have exceeded the max length, %s characters, for the Product Name %s.
-listing.vendorVersion.maxlength=You have exceeded the max length, %s characters, for the Product Version %s.
+listing.DeveloperVersion.maxlength=You have exceeded the max length, %s characters, for the Product Version %s.
 listing.certificationEdition.maxlength=You have exceeded the max length, %s characters, for the Certification Edition %s.
 listing.acbCertificationId.maxlength=You have exceeded the max length, %s characters, for the ACB Certification ID %s.
 listing.certifyingAcb.maxlength=You have exceeded the max length, %s characters, for the Certifying ACB %s.
 listing.testingAcl.maxlength=You have exceeded the max length, %s characters, for the Testing ACL %s.
-listing.vendorStreetAddress.maxlength=You have exceeded the max length, %s characters, for the Vendor Street Address %s.
-listing.vendorStreetAddressTwo.maxlength=You have exceeded the max length, %s characters, for the Vendor Street Address Two %s.
-listing.vendorState.maxlength=You have exceeded the max length, %s characters, for the Vendor State %s.
-listing.vendorCity.maxlength=You have exceeded the max length, %s characters, for the Vendor City %s.
-listing.vendorZip.maxlength=You have exceeded the max length, %s characters, for the Vendor Zip %s.
-listing.vendorWebsite.maxlength=You have exceeded the max length, %s characters, for the Vendor Website %s.
-listing.vendorEmail.maxlength=You have exceeded the max length, %s characters, for the Vendor Email %s.
-listing.vendorPhone.maxlength=You have exceeded the max length, %s characters, for the Vendor Phone %s.
-listing.vendorContactName.maxlength=You have exceeded the max length, %s characters, for the Vendor Contact Name %s.
-listing.vendorState.maxlength=You have exceeded the max length, %s characters, for the Vendor State %s.
+listing.DeveloperStreetAddress.maxlength=You have exceeded the max length, %s characters, for the Developer Street Address %s.
```

Comment:
When I looked at the code before, I thought that the error message was looked up dynamically using a property of the class like "listing." + propertyName + ".maxlength" and I didn't like that because I thought what if you change the property name then the error message will stop showing up and it would be super easy to forget to change the error message name. This is a good illustration of how that can happen. I don't think this change will end with the error message actually showing up because the property on the java object isn't called "DeveloperStreetAddress" (at a minimum the case is not correct, not sure if it's called developer or vendor in there though). I don't have a better suggestion other than to not look up the error property names dynamically.... I don't think doing it that way is beneficial for any reason aside from less typing.

Reply:
Hm, I see what you're saying. I did run the tests locally and they passed, not sure if they'll pass on Jenkins, though. Even if they are not dynamically generated, wouldn't we have to go into the code and change the name as well?

Reply:
If this isn't a good time/ticket for making a refactoring type change I think at least if you're changing the error message names you have to change where they are referenced in the java code. I didn't see anything changed where "vendorWebsite" was referenced, for example, but it got changed to "DeveloperWebsite" below.

Reply:
And I don't see any unit tests written for gov.healthit.chpl.validation.pendingListing.reviewer.FieldLengthReviewer which is where some of these are referenced.

Reply:
Fixed the property names in the Java codes and added the field length reviewer tests, all tests pass on Jenkins as well.